modules/ROOT/pages/dataweave-share-reuse.adoc

```diff
@@ -0,0 +1,18 @@
+= Sharing and Reusing Custom DataWeave Modules and Mappings
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+DataWeave enables you to implement, test, and package your custom DataWeave modules and mappings into libraries that you can share and reuse through Anypoint Exchange.
+
+xref:dataweave-extension-plugin.adoc[DataWeave 2.0 extension to Visual Studio Code] enhances productivity when you are building DataWeave libraries. The extension provides the following capabilities:
+
+* Ability to work on your DataWeave modules and mappings in a standalone manner, outside of a Mule application.
+* Instant execution of your DataWeave mappings, including live preview of data transformations.
+* Language support, including autocompletion, navigation, refactoring, formatting, quick fixes, syntax highlighting, and more.
+* Debugging support for DataWeave mappings.
+* Automatic generation of a documentation template that you can use to provide your own xref:dataweave-create-module.adoc#docs-dw-lib[DataWeave library documentation] in Exchange.
+* An embedded xref:dataweave-maven-plugin.adoc[DataWeave Maven plugin] with the following features:
+** Integration of packaging and deployment of your DataWeave libraries with your Maven lifecycle.
+** Integration with Exchange to share and consume DataWeave libraries.
+* An embedded xref:dataweave-testing-framework.adoc[DataWeave testing framework] that enables you to build automated unit and integration tests easily.
```

Comment:
I would put this item (of DW testing framework) before the item of debugging. Just to put it in some kind of order based on its use in the development life cycle.

Reply:
Yeah, I see your point. I put them in a different order because of the documentation bullet comes before the Maven plugin, both of which talk about Exchange. Bullets are supposed to represent unordered lists, but people might think of them as order.